products/memcache/api.yaml

```diff
@@ -92,6 +93,35 @@ objects:
             - !ruby/object:Api::Type::String
               name: 'message'
               description: The message to be displayed to a user.
+      - !ruby/object:Api::Type::Array
+        name: 'memcacheNodes'
+        description: |
+          Additional information about the instance state, if available.
+        output: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: 'nodeId'
+              description: Identifier of the Memcached node. 
+                The node id does not include project or location like the Memcached instance name.
+            - !ruby/object:Api::Type::String
+              name: 'zone'
+              description: Location (GCP Zone) for the Memcached node.
+            - !ruby/object:Api::Type::Integer
+              name: 'port'
+              description: The port number of the Memcached server on this node.
+            - !ruby/object:Api::Type::String
+              name: 'host'
+              description: Hostname or IP address of the Memcached node used by the clients to connect to the Memcached server on this node.
+            - !ruby/object:Api::Type::String
+              name: state
+              description: |
+                The major version of Memcached software. If not provided, latest supported version will be used.
+                Currently the latest supported major version is MEMCACHE_1_5. The minor version will be automatically
+                determined by our system based on the latest supported minor version.
+              # values:
```

Comment:
Your instinct was right here- it's not worth doing enums for output-only fields. There isn't really an enum type in Terraform- they're just strings with ValidateFuncs to ensure that bad values fail at plan-time. We've added it as a separate type to MM for convenience, but it only makes sense for fields that can actually be set.

Reply:
fixed